implementations/rust/ockam/ockam/src/lib.rs

```diff
@@ -129,4 +129,7 @@ pub mod authenticated_storage {
 }
 
 #[cfg(feature = "ockam_transport_tcp")]
-pub use ockam_transport_tcp::{TcpConnectionTrustOptions, TcpListenerTrustOptions, TcpTransport};
+pub use ockam_transport_tcp::{
```

Comment:
@etorreborre do you think we could create submodule `ockam::tcp`?

Reply:
For consistency that would be great!